contracts/IExtendedERC20.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IExtendedERC20 is IERC20 {
```

Comment:
Could use the existing `IERC20Metadata` interface in extension package.

Reply:
Solved